packages/named-locks/src/index.ts

```diff
@@ -1,17 +1,24 @@
-// @ts-check
-const util = require('util');
+import util from 'util';
+import { PostgresPool, PoolClient } from '@prairielearn/postgres';
+import { PoolConfig } from 'pg';
 
-const { loadSqlEquiv, PostgresPool } = require('@prairielearn/postgres');
-
-const sql = loadSqlEquiv(__filename);
+interface Lock {
+  client: PoolClient;
+}
 
-/** @typedef {import('@prairielearn/postgres').PoolClient} PoolClient */
-/** @typedef {{ client: PoolClient }} Lock */
+interface LockOptions {
+  /** How many milliseconds to wait (anything other than a positive number means forever) */
+  timeout?: number;
+}
 
-/**
- * @typedef {Object} LockOptions
- * @property {number} [timeout] - How many milliseconds to wait (anything other than a positive number means forever)
- */
+type InternalLockOptions =
+  | {
```

Comment:
What does the leading `|` do here?

Reply:
This is just a Prettier thing - I know it looks a little strange! It's equivalent to if there was no leading `|`.